app/services/garden_service.rb

```diff
@@ -0,0 +1,16 @@
+class GardenService
+  def self.garden_details(params)
+    get_parsed_json("api/v1/gardens/#{params[:id]}")
+  end
+
+  def self.get_parsed_json(url, params = {})
+    response = conn.get(url) do |req|
+      req.params = params
```

Comment:
Why do we set the request params equal to params?

Reply:
It would allow us to use the `get_parsed_json` method for API calls that require params like a search call for example. We would pass the query in the params. 

For this specific case that params value isn't really used. It's just the default value - an empty hash.

Reply:
Ahhh I've never seen that before. Cool!